### PR TITLE
fix: [PL-28280]: fixed audit trails proxy url in dev mode

### PR DESCRIPTION
**Pull request by mayankVermaHarness**
### Summary

Fixed audit trails proxy url in dev mode

#### Screenshots

BEFORE:
image

AFTER:
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_NA_** 
 


            
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11284)

**Review by avinashmadhwani02 — APPROVED** [review]

File: configs/devServerProxy.config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-const baseUrl = process.env.BASE_URL ?? 'https://qa.harness.io/gateway'
+const baseUrl = process.env.BASE_URL ?? 'https://app.harness.io/gateway'
```
Comment (upadhyay-an) [inline configs/devServerProxy.config.js]: intended change?

**Review by upadhyay-an — DISMISSED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Comment by mayankVermaHarness** [comment]
retrigger typecheck

**Comment by mayankVermaHarness** [comment]
retrigger jest

**Review by reetika04 — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by upadhyay-an — APPROVED** [review]

**Pull request merged**